back/src/forms/resolvers/mutations/__tests__/markAsSealed.integration.ts

```diff
@@ -84,7 +84,8 @@ describe("Mutation.markAsSealed", () => {
           emitter: CompanyType.PRODUCER,
           recipient: CompanyType.WASTEPROCESSOR,
           trader: CompanyType.TRADER,
-          transporter: CompanyType.TRANSPORTER
+          transporter: CompanyType.TRANSPORTER,
+          broker: CompanyType.BROKER
```

Comment:
c'est lié à quoi cette modification ?

Reply:
Mentionné dans le changelog de la 5.3, set check qu'il n'y ait pas d'undefined desormais

Reply:
pour plus de contexte car c'est vrai que ca n'apparait pas dans le code modifié, l'update avec ça se mettait à planter puisque `companyType(broker)` retournait `undefined`:
```
{
        companyTypes: { set: [companyType(role) as CompanyType] }
}
```

Et le changelog dit ca:
> Validation for undefined values in arrays in Json fields
We added runtime validation for undefined values in arrays in Json fields. Prisma Client will now return an error when an array contains an undefined value. Therefore, we encourage you to add validation that either removes the value or transforms it to null if you stumble on the runtime validation